zip-1337.md

```diff
@@ -0,0 +1,156 @@
+
+Motivation
+==========
+
+Zcash won't thrive without a dev fund. I wish this wasn't true (I really do), and for the longest time I was against the idea. But I've come to fear the alternative without one; I fear the privacy technology pioneered by Zcash may never reach its true potential — not just for our community, but for others interested in novel approaches to private money.
+
+The Foundation, ECC, and broader community has offered many suggestions and guidelines for a potential dev fund, below is my attempt at synthesizing them into a compromise that's greater than the sum of its parts:
+
+- The ECC and Zcash Foundation shouldn't get a blank check; accountability is a prerequisite for any disbursement, based on the [Foundation's statement](https://www.zfnd.org/blog/dev-fund-guidance-and-timeline/) and other proposals being suggested.
+- It's possible for the ECC to remain a for-profit, but with (legally enforced) restrictions that ensure accountability and add teeth to their claim that no early investors are enriched by a new dev fund / no new investors are beneficiaries.
+- Based on the [ideas brought forward by Eran Tromer](https://forum.zcashcommunity.com/t/how-to-hire-ecc/34379/55), a portion of the funds directed to the ECC / Zcash Foundation should be based on shielded adoption, which is the most pressing issue facing Zcash today and unlike many other metrics is both easily measurable and less likely to be gamed. (compared to a miner-signaling proposal for example)
+- A nontrivial portion of the funds should be directed to users/orgs outside of the ECC/Zcash Foundation, and the ECC/Zcash Foundation should be in the minority in deciding how these funds are disbursed (e.g. through some process with broader input beyond ECC/Zcash Foundation employees, like a more constrained version of Placeholder or Blocktower's "third party" proposal).
+- The actual code changes for the NU4 network upgrade should be minimal and the "governance complexity" should be offloaded to legal agreements, not engineering hours. The dev fund would be deposited into a single address for the fund (ideally shielded with a viewing key) controlled through a trust (originally Andrew Miller’s idea), disbursed quarterly based on the accountability requirements and shielded adoption metrics described below. Trustees will be mutually agreed upon by the ECC and Zcash Foundation.
+- The gross amount of the dev fund should still be 20% of the block reward, and it should end in 4 years. (Unless we go through another process like this one to extend it, though I personally hope we don’t)
```

Comment:
> (Unless we go through another process like this one to extend it, though I personally hope we don’t)

What makes you think Zcash won't go through another process like this in four years? If anything, the dependency of the ecosystem on a dev fund will be _even greater_ than it is today.

Reply:
You're right, it's possible that we will/possible that there's a greater dependency at that point. That I'd hope that both the ECC/ZF would be smaller beneficiaries at that point.